src/Commands/generate/Helper/InputHandler.php

```diff
@@ -89,17 +89,16 @@ protected function preprocessQuestions(array &$questions, $destination)
             // Name question.
             if (isset($questions['name'])) {
                 $questions['name']->setAutocompleterValues($modules);
-                $default_name = function ($vars) use ($root_directory) {
-                    return Utils::machine2human(isset($vars['machine_name']) ? $vars['machine_name'] : $root_directory);
+                $default_name = function () use ($root_directory) {
+                  return $root_directory ? Utils::human2machine($root_directory) : '';
```

Comment:
Is the anonymous function needed? How about
```
$default_name = $root_directory ? Utils::human2machine($root_directory) : '';
```

Reply:
Surely, it is not needed now.